app/views/ai-league/ClanSelectorTeachers.vue

```diff
@@ -33,10 +34,24 @@ export default {
       default: () => false
     }
   },
-
+  data () {
+    return {
+      ozClassrooms: []
+    }
+  },
   computed: {
     clansSanitized () {
-      return this.clans.filter(v => v !== undefined)
+      return this.clans.filter(v => v !== undefined).filter(v => !this.ozClassroomsSlugs.includes(v.slug))
```

Comment:
Is it possible that there are `undefined` values among `clans`? why?

Reply:
it has. i don't know why. my testing account has the `undefined` in clans array so the original code is reasonable 😂